native/main.ts

```diff
@@ -145,50 +145,25 @@ const openExternalLink = (url: string): void => {
   }
 };
 
-const startProxy = () => {
-  if (isDev) {
-    return;
-  }
-
-  proxyProcess = execFile(proxyPath, [], (error, stdout, stderr) => {
-    let status = null;
-    let signal = null;
-    if (error) {
-      status = error.code === undefined ? null : error.code;
-      signal = error.signal === undefined ? null : error.signal;
-    } else {
-      status = 0;
-    }
-
-    console.error(
-      "Proxy process exited with status code %s and signal %s",
-      status,
-      signal
-    );
+app.on("will-quit", () => {
+  proxyProcessManager.kill();
+});
 
+// This method will be called when Electron has finished
+// initialization and is ready to create browser windows.
+// Some APIs can only be used after this event occurs.
+app.on("ready", () => {
+  proxyProcessManager.run().then(({ status, signal, output: stdout }) => {
     windowManager.sendMessage({
       kind: MainMessageKind.PROXY_ERROR,
       data: {
         status,
         signal,
         stdout,
-        stderr,
+        stderr: "",
```

Comment:
Dev leftover?
I don't see the proxy error logs anymore when I kill the proxy.

<img width="1552" alt="Screenshot 2020-11-26 at 13 18 26" src="https://user-images.githubusercontent.com/158411/100350367-12407900-2fea-11eb-9b82-29f917e28a94.png">